src/nyc_trees/js/src/searchController.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+
+/* global console */
```

Comment:
Consider adding this to our allowed globals in `src/nyc-trees/.jshintrc` instead of putting this comment here